src/components/LazyLog/LazyList.js

```diff
@@ -164,7 +165,7 @@ export class LazyList extends React.PureComponent {
         selectable={this.props.selectableLines}
         highlight={this.state.highlight.includes(number)}
         onLineNumberClick={this.handleHighlight.bind(this, number)}>
-        {ansiparse(decode(this.state.lines.get(index)))}
+        {line ? ansiparse(decode(line)) : ''}
```

Comment:
Does this cry if you return `null` instead of an empty string? I'd like to not have to allocate a text node if possible.